addons/crm_project_issue/views/crm_lead_views.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <!-- inherit view of Lead Form : adding the "convert to issue" button -->
+    <record id="crm_case_form_view_leads_project_issue" model="ir.ui.view">
+        <field name="name">crm.lead.form.inherit.lead2projectissue</field>
+        <field name="model">crm.lead</field>
+        <field name="inherit_id" ref="crm.crm_case_form_view_leads" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
+                    <button class="oe_stat_button" type="action" name="%(convert_lead2projectissue_wizard_action)d" icon="fa-bug" help="Convert to Issue">
+                        <div class="o_form_field o_stat_info">
+                            <span class="o_stat_text">Convert To</span>
+                            <span class="o_stat_text">Issue</span>
+                        </div>
+                    </button>
+            </xpath>
+        </field>
+    </record>
+
+</odoo>
```

Comment:
@rpi-odoo  can you please tell me what is the need of this blank line ??
its nothing just increase LOC.